js/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Since you depend on native bigint, you should have `engineStrict` set to greater than or equal to node v10 because that was the version they were added. Should also add that to the readme because people don't always look at the `engine` field in the `package.json`

Reply:
`{
  "engineStrict": true,
  "engines": {
    "node": "^10.0.0"
  },` Does that look correct?